### ISSUE TITLE
amendments suggestion

### ISSUE THREAD

**Issue by tamuhey** [ines/spacy-course#47]
*This is a draft, and I will continue to update it as needed.*
 
- [x] "ID" may be a little bit confusing.

https://github.com/ines/spacy-course/blame/master/chapters/en/slides/chapter1_02_statistical-models.md#L104

- [x] not `method`, but `function`?

https://github.com/ines/spacy-course/blame/master/chapters/en/slides/chapter1_02_statistical-models.md#L192

- [x] add backquotes to `words` and `spaces`? (inconsistent to L113)

https://github.com/ines/spacy-course/blame/master/chapters/en/chapter2.md#L127

- [x] not "shared `nlp` object", but "shared `nlp.vocab` object"?

https://github.com/ines/spacy-course/blame/master/chapters/en/chapter2.md#L155

- [x] check if `index < len(doc) - 1`?

https://github.com/ines/spacy-course/blame/master/chapters/en/chapter2.md#L196

- [x] not `one or more`, but `zero or more`

https://github.com/ines/spacy-course/blame/master/chapters/en/slides/chapter2_02_data-structures-2.md#L51

- [x] what this "easier" means? In terms of the computational cost?

https://github.com/ines/spacy-course/blame/master/chapters/en/slides/chapter4_02_training-loop.md#L32

**Comment by ines** [comment]
Thanks! 🎉

> not method, but function?

I've changed it to say "the spacy.explain method". We're typically referring to things like `spacy.load` as methods (even though they're technically functions), so at least this makes it consistent.

> not "shared nlp object", but "shared nlp.vocab object"?

This is supposed to mean that the variable `nlp` has already been created. This was kind of a leftover from the original course materials because they typically stated what objects were already created or preloaded. And I guess it's pointed out in 2.6 because it's one of the first exercises where `nlp` is already defined in the code and not something the user has to write.

If you find it too confusing, I'd just take the whole sentence out.

> check if `index < len(doc) - 1`?

To keep things simple, I didn't include that in the example. But it's mentioned in the success message of that exercise:

https://github.com/ines/spacy-course/blob/a8ab486f5a8425347e51e04f6ee4dffe0b0bdbbe/exercises/en/test_02_07.py#L10-L15

>  not one or more, but zero or more

Technically, yes, but maybe that's a bit confusing? While it's theoretically possible to create a `Span` object consisting of 0 tokens, it's not something you typically would or should do. And if we're talking about a "span" as in a "text span" (not just the `Span` instance), I do think we should mean 1+ tokens.

> what this "easier" means? In terms of the computational cost?

It's more like, the output is more likely to be correct. I actually just changed that part to: "This increases the reliability of the gradient estimates." 

(We tried to keep that simple because we didn't want the course to go too much into arbitrary ML details.)
